src/v2/Artsy/Analytics/Schema/index.ts

```diff
@@ -18,6 +18,14 @@ import { AuthenticationInteraction, Interaction } from "./Interaction"
 import { Label } from "./Label"
 import { Failure, Success } from "./Result"
 import { Type } from "./Type"
+import {
+  ClickedArtistGroup,
+  ClickedArtworkGroup,
+  ClickedAuctionGroup,
+  ClickedCollectionGroup,
+  ClickedFairGroup,
+} from "@artsy/cohesion"
+import { ActionType } from "/Users/ashleyjelks/Code/force/src/v2/Artsy/Analytics/Schema/Values"
```

Comment:
`ActionType` import is pointing to local - should be able to add to the import on the line above.